### ISSUE TITLE
Question about editing files through web UI

### ISSUE THREAD

**Issue by jakepbrown27** [devryan/GamePanelX-V3#162]
Hi Folks - I got this up and running and added a 7 Days to Die template and got it all working!! Loving this so far. I have a question in regards to editing files through the web UI such as in the case of 7D2D the serverconfig.xml I can't seem edit it through the UI but can other text based documents, is it possible to do this? Reason I ask is that I can't find documentation for startup parameters for 7D2D that would override the serverconfig.xml configurations..... Hope that makes sense? 

Also side question: if someone wanted to help maintain this - what would be the best way to go about learning how it works? seems like a JavaScript on the web front end and php in the back?

**Comment by devryan** [comment]
Yep, simple html/javascript with ajax requests on the front end, php 5 on the backend.  It needs to be updated to a newer setup for php 7.

Does it flat out not let you edit the XML file, or it errors out trying?

CPU/RAM checks: are you using local or remote mode?

**Comment by jakepbrown27** [comment]
For the CPU/RAM question: I was using the remote. I had one master (where I presume all the bits are stored) and then the slave running the 7D2D server.  

In regards to the .xml is doesn't seem to give me the option at all - where some files seem to be editable and others are not. I will hop on later and see if I can get a screenshot to better illustrate what I mean.

**Comment by ethsmith** [comment]
@devryan I was planning to work on PR's for this but then I found Go (aka Golang) and it has become my favorite programming language for its performance and syntax and so I decided I was gonna try writing a panel from scratch based off this one. I called it "GoPanelX" haha. I will credit you in the README.

**Comment by devryan** [comment]
@TekkitCommando good luck.  It takes a lot of time/dedication since there are so many random little features that are needed, even before you're actually starting/stopping servers :+1: 

@jakepbrown27 alright.  XML should be editable, let me check on that
